src/files/XmlFile.ts

```diff
@@ -84,20 +45,19 @@ export class XmlFile {
      */
     private unsubscribeFromDependencyGraph: () => void;
 
-    /**
-     * If the file was given type definitions during parse. XML files never have a typedef
-     */
-    public hasTypedef = false;
-
     /**
      * The range of the component's name value
      */
-    public componentNameRange: Range;
+    public get componentNameRange(): Range {
```

Comment:
This was only here because the previous XML parser didn't support locations. I think it makes more sense to store some type of token for `componentName`, and then you can do `xmlFile.componentName.range` and `xmlFile.componentName.text`.

Reply:
Or, we might not even need `XmlFile.componentName` anymore since `parser.references` probably already stores it?